PolyEngine/UnitTests/Src/EnumUtilsTests.cpp

```diff
@@ -5,15 +5,17 @@
 
 using namespace Poly;
 
-enum class eTestEnum
+enum class eTestEnum : uint8_t
```

Comment:
I discussed this on Slack. It's a failed attempt to enforce the enum values to remain in the indexing range (for EnumArray). I don't think it's actually doable without code-gen, though :|
Right now it kind of works as a reminder, but I'm not opposed to removing it.